app/assets/stylesheets/sign_in.scss

```diff
@@ -1,6 +1,6 @@
 body#sign-in {
-  background-color: $primary-bg-color;
-  background: image_url('sign_in_bg.png');
+  background: image_url('sign-in-bg.png');
```

Comment:
Name of function `image_url` should be written in all lowercase letters with hyphens instead of underscores<br>Prefer double-quoted strings